.bacon.yml

```diff
@@ -92,21 +92,20 @@ test_suites:
       sort_order: '13'
       timeout: '15'
       script_name: tsd
-      criteria: OPTIONAL
+      criteria: MERGE
       queue_name: small
-    # FIXME: revert criteria to "MERGE" when merging 0.0 to master
+      # FIXME: revert criteria to "MERGE" when merging 0.0 to master
```

Comment:
Will we do this later? Is there a ticket to track it?

Reply:
Yeah I am going to re-enable this when I merge master and 0.1 together, which will be in this same ticket, actually. I just wanted to keep this PR small